src/main/controllers/dataset_exists.ts

```
@@ -1,28 +1,27 @@
-            if(draftDataset[0].status == 'fulfilled') {
+
+            const draftDataset = await Promise.allSettled([fetchDraftDataset({ datasetId })])
+            if (draftDataset[0].status === 'fulfilled') {
                 return response.status(200).json(draftDataset[0].value)
             }
 
-            if(draftDataset[0].status == 'rejected') { 
-                return response.status(_.get(draftDataset[0], ['reason', 'status'])).json(_.get(draftDataset[0], ['reason', 'response', 'data']))
+            if (draftDataset[0].status === 'rejected') {
+                const errorData = _.get(draftDataset[0], ['reason', 'response', 'data']);
+                return response.status(_.get(draftDataset[0], ['reason', 'status'])).json(errorData)
```

ESLint

[error] Insert `;`

(prettier/prettier)